CONTRIBUTING.md

```diff
@@ -70,7 +70,37 @@ To publish packages to a local registry, do the following:
 - Run `cd ./tmp` in Terminal 2
 - Run `npx create-nx-workspace@999.9.9` in Terminal 2
 
-If you have problems publishing, make sure you use Node 14 and NPM 6 instead of Node 15 and NPM 7.
+If you have problems publishing, make sure you use Node 16 and NPM 6 or 8.
+
+### Publishing for Yarn 2+ (Berry)
+
+Yarn Berry operates slightly differently than Yarn Classic. In order to publish packages for Berry follow next steps:
+
+- Run `yarn set version berry` to switch to latest Yarn version.
+- Create `.yarnrc.yml` in root with following contents:
+  ```yml
+  nodeLinker: node-modules
```

Comment:
Do you have plans to support the default install strategy (`pnp`)? If not, what were the blockers?

Reply:
We do. There are some parts of our codebase that explicitly look for `node_modules` folder and expect to see all packages there, so those would have to be resolved first.

But before all that, we still need to add support for Berry to our CI workflow generators.